frontend/packages/console-dynamic-plugin-sdk/docs/api.md

```
@@ -2997,6 +2997,15 @@ const PodLabelsButton = ({ pod }) => {
 | Parameter Name | Description |
 | -------------- | ----------- |
 | `resource` | The resource to edit labels for, an object of K8sResourceCommon type. |
+| `onSubmit` | Optional callback to customize submit behavior. When provided,  it **completely replaces** the default `k8sPatchResource` JSON patch, including template-selector
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)